src/components/AddExercise.vue

```diff
@@ -441,14 +484,47 @@ export default defineComponent({
       return null;
     },
     enabled(): Exercise[] {
-      return this.exercises.filter(e =>
-        e.trainingDevices.every(t => this.equipments.find(e => e.id == t) && !this.equipments.find(e => e.id == t)?.disabled)
-      );
+      return this.exercises
+        .filter(e => e.trainingDevices.every(t => this.equipments.find(e => e.id == t) && !this.equipments.find(e => e.id == t)?.disabled))
+        .sort(function (a, b) {
```

Comment:
replace function with `(a,b)=> a.name < b.name ? -1 : 1`thtas equivalent and much shorter

Reply:
done

Reply:
ja ne, nix done, auch als arrow function